test/Frontend/skip-non-inlinable-function-bodies.swift

```diff
@@ -157,6 +157,18 @@ public struct Struct {
     }
   }
 
+  public var inlinableWillSet: Int = 1 {
+    willSet {
+      _blackHole("willSet body") // CHECK-NOT: "@inlinable willSet body"
+    }
+  }
+
+  public var inlinableDidSet: Int = 1 {
+    didSet {
```

Comment:
Hm, is this test right? We should be checking that `"didSet body"` _was_ emitted.

Reply:
Yeah, test is very bad. Thanks for pointing out - I originally had these as @inlinable but since they're private that doesn't work. Seems like I forgot to update the name + CHECK though...

Reply:
To clarify this though - we should still be checking that it isn't emitted. It is now SILgen'd (since it's typechecked), but `SILSerializer::shouldEmitFunctionBody` will still not serialize it as it isn't externally visible.